### PR TITLE
Add autocommit to orangelight configs

### LINKED ISSUE #220: Add autocommit to catalog-production and catalog-staging
There's no autocommit. This will be needed when bulk indexing coins.

Here's the one from dpul
https://github.com/pulibrary/pul_solr/blob/master/solr_configs/dpul/conf/solrconfig.xml#L13-L27

### PR DESCRIPTION
**Pull request by hackartisan**
closes #220

### CODE REVIEW

File: solr_configs/catalog-production-v2/conf/solrconfig.xml [hunk 1]
```diff
@@ -35,6 +35,16 @@
     <updateLog>
       <str name="dir">${solr.core0.data.dir:}</str>
     </updateLog>
+    <!-- Hard commit every 60 minutes -->
+    <autoCommit>
+      <maxDocs>10000</maxDocs>
+      <maxTime>36000000</maxTime>
+      <openSearcher>false</openSearcher>
+    </autoCommit>
+    <!-- Soft commit every 5 minutes -->
```
Comment (christinach) [inline solr_configs/catalog-production-v2/conf/solrconfig.xml:44]: Have we tested the autocommit of every 5 min in other apps? Will it maybe increase the CPU?

Comment (hackartisan) [reply solr_configs/catalog-production-v2/conf/solrconfig.xml:44]: Most of the other apps are configured this way. I lifted it from DPUL. I think 5 minutes is reasonable; feel free to ask around for other opinions!

Comment (tpendragon) [reply solr_configs/catalog-production-v2/conf/solrconfig.xml:44]: It should be fine, soft commits are very light and don't do a lot of writes.

Comment (christinach) [reply solr_configs/catalog-production-v2/conf/solrconfig.xml:44]: That's fine. We can always change it back if there is a need to.

**Review by christinach — APPROVED** [review]

**Pull request merged**